src/foam/dao/EasyDAO.js

```diff
@@ -325,20 +301,20 @@ return delegate;
       value: 'id'
     },
     {
-      /** The property on your items to use to store the sequence number or guid. This is required for .seqNo or .guid mode. */
+      documentation: 'The property on your items to use to store the sequence number or guid. This is required for .seqNo or .guid mode',
       name: 'seqProperty',
       generateJava: false,
       class: 'Property'
     },
     {
-      /** Enable local in-memory caching of the DAO. */
+      documentation: 'Enable local in-memory caching of the DAO',
       class: 'Boolean',
       name: 'cache',
       generateJava: false,
       value: false
     },
     {
-      /** Enable standard authorization. */
+      documentation: 'Enable standard authorization',
```

Comment:
It's not necessarily standard anymore, it might use `AuthorizableAuthorizer` if the model implements `Authorizable`. So this should just be `Enable authorization`.